buildozer/default.spec

```diff
@@ -99,6 +99,9 @@ fullscreen = 0
 # (str) Android NDK version to use
 #android.ndk = 9c
 
+# (int) Android NDK API to use (optional). This is the minimum API your app will support. 
+#android.ndk.api = 19
```

Comment:
Just one more minor thing, could you make it `android.ndk_api` rather than `android.ndk.api`? I think the former is more consistent with the other spec lines. Sorry about being picky, but the spec api becomes rather set in stone once release because people start depending on it, so it's worth making sure to get it right.

Reply:
Done!